vignettes/test.Qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "Test"
+author: "dave-lovell"
+format: html
+editor: visual
+---
```

Comment:
Adding this vignette is unnecessary. 

For testing purposes, you can probably use- `tests/testthat/knitr_formats/test.qmd`.

Reply:
This just seemed like the easiest implementation since the tests in `test-lint_dir.R` already use `test_path("dummy_packages", "package", "vignettes")` for the "lint all files in a directory" test.

Reply:
Oh wait are you saying it would be simpler to copy the file you've mentioned into this directory? Got it.

Reply:
I've implemented this in f96b8b0